internal/cmd/serve_command.go

```diff
@@ -109,8 +109,11 @@ func (c *ServeCommand) Run(args []string) int {
 	}
 
 	srv := langserver.NewLangServer(ctx, handlers.NewSession)
-	srv.SetLogger(logger)
-
+	// TODO: Not sure about removing this here. We can't use window/LogMessage yet
+	// because we haven't started
+	// If we leave this here, some stuff still goes to stderr
+	// srv.SetLogger(logger)
```

Comment:
Yes, this is one of the concerns I mentioned in https://github.com/hashicorp/terraform-ls/issues/1271

IMHO I think we'll just need to review what those log messages are, maybe remove any non-critical ones and continue sending the rest to STDERR as there just isn't any better location.